oga/frontend/src/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './App.css';
+import Login from './containers/Login/Login';
+import QuestionList from './containers/QuestionList/QuestionList';
+import NewQuestion from './containers/QuestionList/NewQuestion/NewQuestion.js';
+import RealDetail from './containers/QuestionList/RealDetail/RealDetail.js'
+import Map from './containers/Map/GoogleMap';
+
+import { Route, Redirect, Switch } from 'react-router-dom';
+import { ConnectedRouter } from 'connected-react-router';
+
+
+          //<Route path='/login' exact render={() => <LogIn title="Log In"/>} />
+          //<Route path='/main/questions' exact render={() => <QuestionList title="QUESTIONS"/>} />
+          //<Route path='/main/questions/:id' exact component={RealDetail} />
```

Comment:
I think removing these lines is better.

Reply:
I just left it for future addition, but looks like tidying it up is better idea.